### DISCUSSION TITLE
Failed to run simulation

### DISCUSSION THREAD

**Discussion by yiyuxingchen** [neperfepx/FEPX#80] [Q&A] [answered]
hello everyone，
When i tried to run the second example from the fepx website, Multiaxial Control with Constant Strain Rate (examples/2_triaxCSR), i failed.

```
 Error  :       . Element:        26837 , determinant:   -3.7699936198732666E-007
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 3 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
```
[run file.txt](https://github.com/acmelab-ua/FEPX/files/8638074/run.file.txt)

```
$ neper -T -n 50 -reg 1 -o simulation
$ neper -M simulation.tess -rcl 0.5 -order 2
$ mpirun -np 4 fepx
version 1.3.1
```

[simulation.config.txt](https://github.com/acmelab-ua/FEPX/files/8638044/simulation.config.txt)

Is there any configuration code for the case on the fepx official website that can help me learn？
Thanks.

**Comment by mkasemer** [comment, marked as answer]
Hello @yiyuxingchen , Unfortunately I am unable to reproduce the error. What are your system specifications (compiler, openMPI version, OS, etc.), and can you please copy the terminal output for your `ccmake` to show what was used during compilation?

**Reply by yiyuxingchen** [reply]
Thanks for your answer. After I enter ccmake in the terminal under the installation directory, the following content appears, what should I do?
![image](https://user-images.githubusercontent.com/103731227/167989087-61ac2c5b-d86b-47c6-825d-256729695023.png)

**Reply by mkasemer** [reply]
@yiyuxingchen: please be sure to copy and paste text rather than screenshots.

Regarding my request: I am specifically asking to see the `ccmake` output when you install FEPX, which will list most things necessary to see your operating environment.

**Reply by yiyuxingchen** [reply]
I think I found the problem, the deformation conditions for multiaxial loads should not be written as,
`    boundary_conditions triaxial
    loading_direction x
    strain_rate -1e-4
    loading_direction z
    strain_rate 1e-4`
but，
`    boundary_conditions triaxial
    loading_direction x
    strain_rate -1e-4`

If so, I have a question, when there are loads in all three directions, is the loading direction defined as x, y or z?

**Reply by mkasemer** [reply]
@yiyuxingchen : Please consult the manual to see the general form of the input, as well as the examples to see a specific example of said input employed in a simulation.

As a note, you can only set one primary loading direction and one primary strain rate which is enforced in that loading direction. Strain rates in the other directions will have to vary in order to hit the target loads specified for each surface.
